setup.py

```diff
@@ -180,6 +180,7 @@
             'jupyter-notebook = notebook.notebookapp:main',
             'jupyter-nbextension = notebook.nbextensions:main',
             'jupyter-serverextension = notebook.serverextensions:main',
+            'jupyter-bundler = notebook.bundlerextensions:main',
```

Comment:
I think I prefer `jupyter <splat>` for consistency with the others. But we can definitely change the name from `bundler` to `bundlerextension` or something more descriptive.